go/tools/bazel/runfiles/directory.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018 Google Inc.
```

Comment:
Please change to `Copyright 2018 The Bazel Authors.`. Same for other new .go files in this PR.

Reply:
Done.